### ISSUE TITLE
DB table and field name convention

### ISSUE THREAD

**Issue by jakubzapletal** [spapali/Blue-Economics#29]
I would be good to have a same naming convention for table names and field names in all tables :-)

It might be good to create a DOC document with documentation for that or use README.md for this reason.

@innls @jemuelyoung @dasibre @rosebery19 @obraafo

**Comment by obraafo** [comment]
Agreed :+1:  We should just agree on a convention and use that. I personally prefer [PascalCase](http://en.wikipedia.org/wiki/CamelCase) although I've actually not been consistent myself with regards to the convention in the new FAQ tables in `faq-app/blueecon-faq-ddl.sql` :)

**Comment by jakubzapletal** [comment]
@obraafo do you really insist on PascalCase in naming strategy of database tables and fields? It's common used in naming of classes but in database it's very rare. I have seen few frameworks and projects but all with underscore naming strategy so it seems to be for me very unusual.

**Comment by obraafo** [comment]
@jakubzapletal I'm not insisting on PascalCase. Just saying that's my preference; Don't see why we need different conventions for database names vs. PHP code for example. It doesn't matter either way as long as we're consistent.

**Comment by jakubzapletal** [comment]
@obraafo now I understand your thoughts. Have you ever seen PascalCase for DB naming in any mass-used projects, maybe any open source? Please don't get it wrong, I only would like to know to improve my knowledge of use cases.

**Comment by raynor85** [comment]
For table naming I prefer singular-lowercase-underscore naming practice, I think that formula is more common, but there isn't a standard way to do that. As long as we are consistent, it's fine.
